lib/common/monitoring/alarms/KinesisAlarmFactory.ts

```diff
@@ -14,6 +14,13 @@ export interface RecordsThrottledThreshold extends CustomAlarmThreshold {
   readonly maxRecordsThrottledThreshold: number;
 }
 
+export interface FirehoseStreamLimitThreshold extends CustomAlarmThreshold {
+  /**
+   * Threshold value between [0.0, 1.0) for when the alarm should be triggered.
```

Comment:
We should probably check for that range within `addFirehoseStreamExceedSafetyThresholdAlarm` to prevent any incorrect usages, along with adding some tests for that.

Reply:
done.